Makefile

```diff
@@ -21,6 +24,11 @@ help: Makefile ## This help
 init:  ## Terraform init
 	@echo "Please refer to the latest instructions and terraform.tfvars.json file format at https://github.com/tetrateio/tetrate-service-bridge-sandbox#usage"
 
+
+.PHONY: validate
+validate:  ## Validate terraform input
+	export TFVARS_JSON=${tfvars_json} && ./make/validate.sh validate_input
```

Comment:
should we make validate required and be part if variables.sh and basically be mandatory for each run, we can have it as a separate make target too, but by default it should run everytime in my opinion as variables.sh is executed.

Reply:
I my avalanche branch, I had it as a prerequisite target on the deploy targets. forcing it to run automatically, yet having the flexibility to run it separately as well (to debug a broken json).